packages/cocoon_common/test/buffered_logger_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_common/cocoon_common.dart';
+import 'package:cocoon_common_test/cocoon_common_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('uses the provided DateTime for recordedAt', () {
+    final moment = DateTime.now();
```

Comment:
we sure we don't just want to use Clock?

Reply:
Go clock yourself.